utils/src/lib.rs

```diff
@@ -1,3 +1,4 @@
+#![allow(dead_code)]
```

Comment:
is this necessary?

Reply:
yeah because of the Debug derives for the private types, but I want this because useful for debugging